.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npx lint-staged
+npm run extract-messages
```

Comment:
maybe put this in the lint-staged configuration and only run when relevant files are staged? e.g.

```json
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": "npm run extract-messages"
  },
```